services/flathub/flathub.tester.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const {
+  isVPlusDottedVersionNClausesWithOptionalSuffixAndEpoch,
+} = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('Flathub (valid)').get('/org.mozilla.firefox.json').expectBadge({
+  label: 'flathub',
+  message: isVPlusDottedVersionNClausesWithOptionalSuffixAndEpoch,
```

Comment:
Do flatpak versions actually have the epoch clause?

Reply:
I copied the code form archlinux 😅